sources/Context.cpp

```diff
@@ -344,6 +345,37 @@ Context::executeSecurityAnalysis() {
   saLauncher->writeResults();
 }
 
+void
+Context::exportResults(bool isOk) {
+  switch (def_.simulationKind) {
+  case SimulationKind::STEADY_STATE_CALCULATION: {
+    boost::property_tree::ptree resultsTree, componentResultsTree, componentResultsChild;
+    resultsTree.put("version", "1.2");
```

Comment:
Is this the powsybl version? If so we are using 1.4

Reply:
No this is the version of the LodFlowResult format. See https://github.com/powsybl/powsybl-core/blob/main/loadflow/loadflow-api/src/test/resources/LoadFlowResultVersion12.json